src/Servers/HttpSys/test/FunctionalTests/ResponseCachingTests.cs

```diff
@@ -21,7 +21,7 @@ public class ResponseCachingTests
 
         public ResponseCachingTests()
         {
-            _absoluteFilePath = Directory.GetFiles(Directory.GetCurrentDirectory()).First();
+            _absoluteFilePath = Path.Combine(Directory.GetCurrentDirectory(), "Microsoft.AspNetCore.Server.HttpSys.dll");
```

Comment:
Hmm, why not a test specific text file that we include as content? That would probably be more stable.

Reply:
*shrug* when we're not interested in the contents of the file then any file will do.

Reply:
That was probably the reasoning behind .First() but apparently that can cause issues.